### PR TITLE
#1298: Continuous deploy after updates on component repos (closes #1298)

### LINKED ISSUE #1298: Continuous deploy after updates on component repos
[Project card](https://github.com/buildo/react-components/projects/1?fullscreen=true#card-16531021)

## stories
As a user i want the `showroom` to be evergreen otherwise I cannot trust what I see

## requirements
Trigger a deploy after anyone of the component's repo has been updated with a new commit on `master`

## specs
...

### PR DESCRIPTION
**Pull request by fes300**
Closes #1298

### tests performed

screenshot 2019-01-15 at 12 30 54

### CODE REVIEW

**Comment by fes300** [comment]
this is still not deploying the artifact (just creating it)

**Comment by fes300** [comment]
my idea is that the `package.json` in the `showroom` repo should point to the `master` branch of our repos, this way it will be very easy to update it

File: ci/deploy.yml [hunk 1]
```diff
@@ -0,0 +1,67 @@
+#######################
+# resources
+#######################
+
+resources:
```
Comment (gabro) [inline ci/deploy.yml]: image

This reminds me of

```java
/**
  * gets the foo
  * @returns a foo of type Foo
  */
  Foo getFoo() {
    return this.foo
  }
```

Comment (fes300) [reply ci/deploy.yml]: hahahah

File: ci/deploy_showroom.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+run:
+  path: sh
+  args:
+    - -c
+    - |
+      set -e
+      cd showroom
+      yarn install --no-progress
+      yarn build
+      echo "now you should put what you just built somewhere.."
```
Comment (gabro) [inline ci/deploy_showroom.yml]: since we're doing this, maybe split this in `deploy_showroom.sh`

**Comment by fes300** [comment]
@gabro are we ready to merge this after our offline discussion?

**Pull request merged**